source/index.html.md

```diff
@@ -350,18 +351,178 @@ curl "https://api.kontist.com/api/accounts/4711/transfer/f55641811042b9e85989bd5
 
 ```json
 {
-    "status": "accepted",
-    "recipient": "Karl Brenner",
-    "iban": "DE89370400440532013000",
-    "amount": 1250,
-    "note": "Thank you for paying my lunch",
-    "meta": {
-        "authorizationTokenLength": 6
-    },
-    "links": {
-        "self": "/api/accounts/4711/transfer/f55641811042b9e85989bd57c3718346ctrx"
-    }
+  "status": "accepted",
+  "recipient": "Karl Brenner",
+  "iban": "DE89370400440532013000",
+  "amount": 1250,
+  "note": "Thank you for paying my lunch",
+  "meta": {
+    "authorizationTokenLength": 6
+  },
+  "links": {
+    "self": "/api/accounts/4711/transfer/f55641811042b9e85989bd57c3718346ctrx"
+  }
+}
+```
+
+# Timed orders
+
+A timed_order is a regular SEPA Credit Transfer which is pre-authorized to be executed once at a specific date.
```

Comment:
is underscore correct there? 🤔I was thinking it's the regular term `timed order` in this sentence

Reply:
good catch blindly copied it from Solaris